src/main/java/uk/gov/justice/hmpps/prison/service/BedAssignmentHistoryService.java

```diff
@@ -87,7 +87,7 @@ public List<BedAssignment> getBedAssignmentsHistory(final long livingUnitId, fin
             .toList();
     }
 
-    @VerifyAgencyAccess(overrideRoles = {"SYSTEM_USER"})
```

Comment:
The endpoint which calls this service is only called by prison-staff-hub which uses AuthorizationCode - so the overrideRoles can safely be removed